src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -1,29 +1,81 @@
 <template>
   <div id="amalgamating-businesses">
-    <v-btn
-      id="add-amalgamating-business-button"
-      outlined
-      color="primary"
-      class="btn-outlined-primary"
-      :disabled="isAddingAmalgamatingBusiness || isAddingAmalgamatingForeignBusiness"
-      @click="isAddingAmalgamatingBusiness = true"
-    >
-      <v-icon>mdi-domain-plus</v-icon>
-      <span>Add an Amalgamating Business</span>
-    </v-btn>
-
-    <v-btn
-      v-if="isRoleStaff"
-      id="add-foreign-business-button"
-      outlined
-      color="primary"
-      class="ml-2 btn-outlined-primary"
-      :disabled="isAddingAmalgamatingBusiness || isAddingAmalgamatingForeignBusiness"
-      @click="isAddingAmalgamatingForeignBusiness = true"
-    >
-      <v-icon>mdi-domain-plus</v-icon>
-      <span>Add an Amalgamating Foreign Business</span>
-    </v-btn>
+    <section class="checklist-section">
+      <div class="subhead">
+        Your application must include the following:
+      </div>
+      <ul>
+        <li v-if="isAmalgamationFilingHorizontal || isAmalgamationFilingVertical">
+          <v-icon
+            v-if="haveHoldingBusiness"
+            color="green darken-2"
+            class="cp-valid"
+          >
+            mdi-check
+          </v-icon>
+          <v-icon
+            v-else-if="getShowErrors"
+            color="error"
+            class="cp-invalid"
+          >
+            mdi-close
+          </v-icon>
+          <v-icon v-else>
+            mdi-circle-small
+          </v-icon>
+          <span class="rule-item-txt">The {{ isAmalgamationFilingHorizontal ? 'primary' : 'holding' }}
+            business</span>
+        </li>
+
+        <li>
+          <v-icon
+            v-if="haveAmalgamatingBusinesses"
+            color="green darken-2"
+            class="incorp-valid"
+          >
+            mdi-check
+          </v-icon>
+          <v-icon
+            v-else-if="getShowErrors"
+            color="error"
+            class="incorp-invalid"
+          >
+            mdi-close
+          </v-icon>
+          <v-icon v-else>
+            mdi-circle-small
+          </v-icon>
+          <span class="rule-item-txt">Amalgamating businesses</span>
+        </li>
+      </ul>
+    </section>
```

Comment:
Example for regular amalgamation:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/e06af640-48ae-442e-88ef-76d76a14daaa)

Reply:
Example for short-form amalgamation:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/16202b6a-ab72-47a9-9af8-1727a06a415b)